kyo-offheap/native/src/main/java/lang/foreign/Arena.scala

```diff
@@ -0,0 +1,26 @@
+package kyo.offheap.native.java.lang.foreign
+
+import scala.scalanative.unsafe._
+import scala.scalanative.libc.stdlib._
+
+// Stub implementation for Java's Arena
+class Arena {
+  // Allocate a memory segment of the specified size
+  def allocate[A: Layout](size: Long): MemorySegment = {
```

Comment:
have you attempted to build and execute the tests? Subs must offer the same APIs as their original counterparts but limited to the ones used by the current implementations.